### PR TITLE
Trn 53 send avatar to django

### PR DESCRIPTION
**Pull request by jestebanpelaez18**
Now we can upload avatar in django. When you test it with postman you should be able to see the media folder in the working directory. Sent the post request like in the attached file. Dont forget to create the user and validate the token.
Screenshot 2024-06-11 at 18 19 22

### CODE REVIEW

**Review by RushMaverick — APPROVED** [review]
I want to try this out on my machine, but it seems to work from what we tested. Approved!

**Review by rosamakinen — APPROVED** [review]

File: backend/django_backend/user/models.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+    return 'images/{filename}'.format(filename=filename)
+
+class Avatar(models.Model):
     user = models.OneToOneField(
         User,
         on_delete=models.CASCADE,
         primary_key=True,
     )
-    # avatar = models.ImageField(upload_to='profile_pics', default='default.jpg')
+    image = models.ImageField(upload_to=upload_to, blank=True, null=True)
```
Comment (Stte) [inline backend/django_backend/user/models.py:13]: I think the default image is needed or how are we setting the default avatar?
"Users can upload an avatar, with a default option if none is provided"

File: backend/django_backend/django_backend/urls.py [hunk 2]
```diff
@@ -37,4 +38,8 @@
     path('admin/', admin.site.urls),
     path('api/token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('api/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
-]
+    path('profile/', include('user.urls'))
+] 
+
+if settings.DEBUG:
```
Comment (Stte) [inline backend/django_backend/django_backend/urls.py]: What do we do in production mode?

**Review by Stte — CHANGES_REQUESTED** [review]
Nice! Upload worked. The uploaded avatars were accessible thorough http://localhost:8000/media/images/[image]

Very good! Just two questions in comments.

File: backend/django_backend/user/models.py (hunk 1 above)
Comment (jestebanpelaez18) [reply backend/django_backend/user/models.py:13]: When i was using the default.jpg, that was causing issues did not let update the image. So I changed to that way,  that means that the form doesn't require a value but the database does.

Comment (Stte) [reply backend/django_backend/user/models.py:13]: Ok so we need to have some sort of front-end solution to show a default avatar if the avatar is not set?

**Review by Stte — APPROVED** [review]
LGTM!

**Pull request merged**